spyder/widgets/reporterror.py

```diff
@@ -345,6 +345,12 @@ def _contents_changed(self):
                               title_chars >= TITLE_MIN_CHARS)
         self.submit_btn.setEnabled(submission_enabled)
 
+    def set_title_description(self, title, description):
+        """Set a title and description to the window."""
+        self.main_label.setText(
+            _("<h3>{title}</h3>"
+              "{description}").format(title, description))
```

Comment:
Ok, this is not what I have in mind. Please change this to

```python
def set_title(self, title):
    self.title.setText(title)

def set_description(self, description):
    self.input_description.setPlainText(description)
```

The idea is to automatically fill the title and description of the report because we already know what this error is about.

Reply:
I understand now what you meant :o

Reply:
Great!!